packages/muelu/src/Transfers/GeneralGeometric/MueLu_RegionRFactory_decl.hpp

```diff
@@ -48,7 +48,7 @@
 #define MUELU_REGIONRFACTORY_DECL_HPP
 
 #include "MueLu_ConfigDefs.hpp"
-#ifdef HAVE_MUELU_KOKKOS_REFACTOR
+#ifdef HAVE_MUELU_TPETRA
```

Comment:
Actually `MueLu_RegionRFactory` really needs the Kokkos guard as it is using the `KokkosSparse::CrsMatrix` to construct the restriction operator.